packages/fluentui/docs/src/routes.tsx

```diff
@@ -149,12 +147,6 @@ const Routes = () => (
                   <MarkdownPage page={ShorthandProps} />
                 </Route>
                 <Route exact path="/icon-viewer" component={IconViewer} />
-                <Route exact path="/theming-specification">
-                  <MarkdownPage page={ThemingSpecification} />
```

Comment:
This document does not have relationship to N* codebase, is visible only locally and is more confusing than useful.